mappings/net/minecraft/block/IBlockInfoProxy.mapping

```diff
@@ -56,9 +56,11 @@ CLASS none/asl net/minecraft/block/IBlockInfoProxy
 		ARG 0 view
 		ARG 1 pos
 	METHOD e allowsLightTransmission ()Z
+	METHOD e getOffsetPos (Lnone/aja;Lnone/cn;)Lnone/bcv;
 	METHOD g getMapColor ()Lnone/ays;
 	METHOD h ()Z
 	METHOD i getRenderType ()Lnone/api;
+	METHOD j getAOLightValue ()F
```

Comment:
Technically this should be `getAoLightValue` (we also use `Gl` and `Uuid` ofr example) but that looks stupid. I'd say write it out: `getAmbientOcclusionLightValue`.